requirements/dev.txt

```diff
@@ -0,0 +1,36 @@
+appdirs==1.4.3
+atomicwrites==1.3.0
+attrs==19.3.0
+certifi==2020.4.5.1
+configparser==4.0.2
+contextlib2==0.6.0.post1
+distlib==0.3.0
+elasticsearch==7.6.0
+entrypoints==0.3
+enum34==1.1.10
+filelock==3.0.12
+flake8==3.7.9
+funcsigs==1.0.2
+functools32==3.2.3.post2
+importlib-metadata==1.6.0
+importlib-resources==1.4.0
+mccabe==0.6.1
+more-itertools==5.0.0
+packaging==20.3
+pathlib2==2.3.5
+pluggy==0.13.1
+py==1.8.1
+pycodestyle==2.5.0
+pyflakes==2.1.1
+pyparsing==2.4.7
+pytest==4.6.9
+scandir==1.10.0
+singledispatch==3.4.0.3
+six==1.14.0
+toml==0.10.0
+tox==3.14.6
+typing==3.7.4.1
+urllib3==1.25.8
+virtualenv==20.0.17
+wcwidth==0.1.9
+zipp==1.2.0
```

Comment:
out of curiosity this is a mix of testing dependencies and elasticstat dependencies correct?

I guess what I'm thinking we ideally want to split these to what elasticstat directly depends on (and have those depencies pinned like `elasticsearch`), dev requirements for dependencies that are needed just for testing but don't need to installed by users.